packages/docs/page-config/ui-elements/data-table/examples/Sticky.vue

```diff
@@ -60,9 +44,7 @@
     >
       <template #headerPrepend>
         <tr>
-          <th colspan="6">
-            With scroll events
-          </th>
+          <th colspan="6">With scroll events</th>
```

Comment:
This is also complicated demo. Why do we need to have infinite scroll in demo related to sticky header? Just show a sticky header and we're done.